src/manifest.c

```diff
@@ -110,7 +110,12 @@ static struct manifest *manifest_from_file(int version, char *component, bool he
 	} else {
 		basedir = MIX_STATE_DIR;
 	}
-	string_or_die(&filename, "%s/%i/Manifest.%s", basedir, version, component);
+
+	if (delta_manifest_version > 0) {
+		string_or_die(&filename, "%s/%i/Manifest.%s.D.%i", basedir, delta_manifest_version, component, version);
```

Comment:
Leave a comment here that delta manifests are extracted to the `delta_manifest_version` version directory (which is the `last_change` version) but have the `version` in the name.